cost-analyzer/values.yaml

```diff
@@ -693,9 +693,15 @@ ingress:
   enabled: false
   # className: nginx
   labels:
+    ## Starting in kubernetes v1.18, `kubernetes.io/ingress.class` annotation is deprecated.
```

Comment:
Not sure if this comment is valid for labels too.

Reply:
@mittal-ishaan I agreed. Resolved at [86c39b9](https://github.com/kubecost/cost-analyzer-helm-chart/pull/3749/commits/86c39b96aa65bd14fe5dd1795794175db28f5f76).